### PR TITLE
5 patch /api/reviews/review

### PR DESCRIPTION
**Pull request by Oathian**

### CODE REVIEW

File: models/reviews.models.js
Comment (elijaharley) [inline models/reviews.models.js]: These errors can be handled as custom errors in the app - eg if (err.code === '22P02') - if it is not a number it will automatically send back an error code which can then be handled rather than checking isNaN in every model. Let me know if you need any help with this.

**Review by elijaharley — CHANGES_REQUESTED** [review]
Hi Aiden, some PSQL error handling will need to be updated before approval - see comment and let me know if you need help with this.

File: models/reviews.models.js
Comment (Oathian) [reply models/reviews.models.js]: Hi Eli, I've had a go at redoing the error handling with PSQL

**Review by elijaharley — CHANGES_REQUESTED** [review]
Hi Aidan, looking good - I'd just reorder those error handlers so that PSQL errors are above custom errors, then if there are PSQL errors you can assume these have been handled - this means you may still need some Promise.reject statements in your models if result.rows.length === 0

**Comment by Oathian** [comment]
> 

Hi Eli, I've reordered the error handlers. Also I think I've already got the Promise.reject for rows.

**Review by elijaharley — APPROVED** [review]
Great stuff Aidan, you're good to merge!

**Pull request merged**
